cmd/notary/cert.go

```diff
@@ -63,29 +64,30 @@ func certRemove(cmd *cobra.Command, args []string) {
 	}
 
 	var certsToRemove []*x509.Certificate
+	var certFoundByID *x509.Certificate
 
 	// If there is no GUN, we expect a cert ID
 	if certRemoveGUN == "" {
 		certID := args[0]
 		// This is an invalid ID
-		if len(certID) != idSize {
+		if len(certID) != notary.IDSize {
```

Comment:
I'm actually convinced this check isn't at all necessary... Regardless of ID size, if a file exists with the ID, we should delete it, if it doesn't we won't. Similar logic applies to other places we have these checks for the ID size.